loky/backend/synchronize.py

```diff
@@ -24,19 +24,26 @@
 from . import resource_tracker
 
 __all__ = [
-    'Lock', 'RLock', 'Semaphore', 'BoundedSemaphore', 'Condition', 'Event'
-    ]
+    "Lock",
+    "RLock",
+    "Semaphore",
+    "BoundedSemaphore",
+    "Condition",
+    "Event",
+]
 # Try to import the mp.synchronize module cleanly, if it fails
 # raise ImportError for platforms lacking a working sem_open implementation.
 # See issue 3770
 try:
     from _multiprocessing import SemLock as _SemLock
     from _multiprocessing import sem_unlink
 except ImportError:
-    raise ImportError("This platform lacks a functioning sem_open" +
-                      " implementation, therefore, the required" +
-                      " synchronization primitives needed will not" +
-                      " function, see issue 3770.")
+    raise ImportError(
+        "This platform lacks a functioning sem_open"
+        + " implementation, therefore, the required"
+        + " synchronization primitives needed will not"
+        + " function, see issue 3770."
```

Comment:
The `+` operators are not needed here I think (because of the surrounding parens). Also it would be worth using fewer lines by packing more words per line.